doc/pipeline-details.md

```diff
@@ -15,15 +15,14 @@ flowchart
     subgraph trigger_service[Trigger Service]
         trigger[Trigger] --> kernel_revision{New kernel <br/>revision ?}
         kernel_revision --> |No| trigger
-        kernel_revision --> |Yes| checkout[Create 'checkout' node <br />state=available, result=None, set holdoff]
+        kernel_revision --> |Yes| checkout[Create 'checkout' node <br />state=running, result=None]
     end
     subgraph tarball_service[Tarball Service]
-        checkout --> |event: <br />checkout created, state=available| tarball[Tarball]
-        tarball --> tarball_node[Create 'tarball' node <br />state=running, result=None, holdoff=None <br />Update 'checkout' node <br />with describe and artifacts]
-        tarball_node --> upload_tarball[Create and upload tarball to the storage <br />state=available, result=None, set holdoff]
+        checkout --> |event: <br />checkout created, state=running| tarball[Tarball]
```

Comment:
What does `tarball[Tarball]` mean here?  I think the block from the `runner` service about uploading the tarball should be moved here instead as that's what the tarball service does.

Reply:
Definitely.
Those are some glitches that may be introduced when I renamed `tarball` node.
Need to fix this.

Reply:
I have created it with mermaid live editor and the block you mentioned is in `tarball` service itself.
I wonder why it is different on staging.
Attaching the flowchart from https://mermaid.live

![image](https://user-images.githubusercontent.com/11505157/201644239-eb916dff-44cb-4802-924a-31a2bf4677f5.png)

Reply:
Even https://github.com/kernelci/kernelci-api/blob/staging.kernelci.org/doc/pipeline-details.md matches with the flowchart I attached.